BAZProject/GenericComponents/Cell/EmptyMoviesCollectionCell.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  EmptyMoviesCollectionCell.swift
+//  BAZProject
+//
+//  Created by 1014600 on 23/02/23.
+//
+
+import UIKit
+
+class EmptyMoviesCollectionCell: UICollectionViewCell {
+
+    static var cellIdentifier: String = "EmptyCollectionCell"
+
+    let nameLabel: UILabel = {
+        let label = UILabel()
+        label.font = UIFont.systemFont(ofSize: 20)
+        label.textColor = .lightGray
+        label.backgroundColor = .white
+        label.text = "Sin resultados"
+        label.textAlignment = .center
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setupCell()
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+    }
+    
+    func setupCell(){
+        addSubview(nameLabel)
+
+        nameLabel.widthAnchor.constraint(equalTo: widthAnchor).isActive = true
+        nameLabel.heightAnchor.constraint(equalToConstant: UIScreen.main.bounds.width - 20).isActive = true
```

Comment:
Para sacarle provecho a los constraints es importante no utilizar (o reducir mucho su uso) valores absolutos que pueden cambiar con el tiempo de vida de tu aplicación. Si rotas el dispositivo `UIScreen.main.bounds.width` va a cambiar y puedes obtener comportamientos no deseados.

```suggestion
nameLabel.heightAnchor.constraint(equalTo: widthAnchor).isActive
```